### ISSUE TITLE
WILCARD_VALUE not in closure

### ISSUE THREAD

**Issue by jdtsmith** [fizista/micropython-scron#3]
I'm getting this error occasionally:

```
Traceback (most recent call last):
  File "/flash/lib/scron/week.py", line 74, in _next_step
  File "/flash/lib/scron/week.py", line 61, in get_next_pointer
  File "/flash/lib/scron/scount.py", line 255, in get_next_pointer
  File "/flash/lib/scron/helpers.py", line 118, in items
AttributeError: 'closure' object has no attribute 'WILDCARD_VALUE'
```

Thanks for scron.

**Comment by fizista** [comment]
Strange error.

On what device, you get that bug?

What version of micropython?

Could you add the following code before the 118 line in scron/helpers.py file:

```
  for attr in dir(self):
    print("obj.%s = %r" % (attr, getattr(self, attr)))
```

Maybe it will tell us more about what is going on there.

**Comment by jdtsmith** [comment]
Thanks.  I'm using [Loboris MP](https://github.com/loboris/MicroPython_ESP32_psRAM_LoBo/wiki) on ESP32.  In fact I found that `scron` does not run well at all under that port on the device, leading, in addition to occasional closure errors, to lots of CPU halts apparently related to memory handling, ala:

```
Guru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.
...
assertion "VERIFY_PTR(ptr)" failed: file "/home/LoBo2_Razno/ESP32/MicroPython/MicroPython_ESP32_psRAM_LoBo/MicroPython_BUILD/components/micropython/py/gc.c", line 594, function: gc_free
abort() was called at PC 0x40117f23 on core 1
...
```
etc.

These halts were sporadic — sometimes immediately on import, sometimes later.  I couldn't identify any particular step or sequence of actions that produced them.  In the end I had to abandon `scron` and write my own simpler loop-polling based scheduler (borrowing the _pointer_ idea, thanks).  This has eliminated all such halts.

As to debugging this, I'd guess the issue will be that the loboris ESP32 MP port is [no longer supported](https://github.com/loboris/MicroPython_ESP32_psRAM_LoBo/issues/302), so if it contains memory handling bugs that `scron` tickles, no one is there to address them.  

Sadly, the loboris port has major functionality that I can't find anywhere else: display support for common packaged displays like `ST7789`, background threaded and efficient FTP client, multiple built-in as well as user-compilable font support, and much more.  Given the development style, sadly there [isn't much appetite](https://github.com/loboris/MicroPython_ESP32_psRAM_LoBo/issues/304) to port any of this to mainline.
